packages/block-editor/src/components/media-upload/index.native.js

```diff
@@ -185,7 +201,7 @@ export class MediaUpload extends Component {
 			multiple = false,
 		} = this.props;
 
-		if ( value === 'URL' ) {
+		if ( value === URL_MEDIA_SOURCE ) {
 			prompt(
```

Comment:
@jd-alexander, I downloaded the Android APK to test and use the opportunity to learn from your code. :) Everything worked smoothly in my testing, I just have a question about the `Type a URL` dialog generated by the `prompt` here. 

Any content added to the dialog is cleared with a screen rotation, which I'm learning happens with a few dialogs, specifically on Android devices. My question is, is fixing smaller platform-specific issues like this generally a judgment call between importance vs. the amount of time/effort it'd take to create a solution? I'm really curious whether this was an issue you'd already spotted, as well as your process around deciding whether it was worth investing in a solution or not.

Reply:
Good eye @SiobhyB thanks for testing and spotting. 

> Any content added to the dialog is cleared with a screen rotation, which I'm learning happens with a few dialogs, specifically on Android devices. My question is, is fixing smaller platform-specific issues like this generally a judgment call between importance vs. the amount of time/effort it'd take to create a solution?

Good question. In a situation like this, the fix isn't as easy as doing it within WP Android because the code for the dialog lives in an external dependency of https://github.com/wordpress-mobile/react-native-prompt-android so we would have to follow a similar strategy as the other `DialogFragments` in WP Android and persist the arguments to `savedInstanceState` that are passed over the bridge to this component. I can create an issue within the component to rectify this as it's not complex to do, but the `Example` app there needs to be updated to make testing possible.